src/Portfolio/Card.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./Card.scss";
+
+const Card = ({ image, imageName, name, desc }) => {
```

Comment:
You can implicitly return here instead of writing return 

E.g. Change the { to a ( and remove line 5